docs/product/explore/profiling/transaction-vs-continuous-profiling.mdx

```diff
@@ -85,3 +85,10 @@ Continuous profiling can give you visibility into parts of your application that
 If you want to profile a high-throughput server with little or no idle time, continuous profiling can help reduce costs, as the profiler will not upload duplicated profiling data from overlapping transactions.
 
 It's important that you understand the tradeoffs between the two modes and choose the one that best fits your use case. If you're unsure, we recommend starting with continuous profiling because you'll get a more complete picture of your application's performance.
+
+
+## Missing Transaction Data
+
+If you have clicked on profiling and landed on a page that displays no data, yet displays a banner that tells you to look at the aggregate flamegraph, it means that Sentry has only received profiling data from your project, but no spans or transactions. This can be a valid use case when profiling is setup, but tracing is not. We discourage this, as tracing data can complement profiling in valuable ways, and unless you are sure what you are doing, we would recommend you to enable tracing as well.
+
+If you have configured tracing and are sending us tracing data, yet you still see this message, please contact us via support.
```

Comment:
```suggestion
## Missing Profiling Data

If your [**Profiling**](https://sentry.io/orgredirect/organizations/:orgslug/profiling/) page displays no profiling data, and you're seeing a "look at the aggregate flamegraph" banner, it means that Sentry has received profiling data from your project, but no [spans or transactions](/product/tracing/). While this can be a valid use case when profiling is set up, but tracing is not [WOULD BE GOOD TO ADD A LINE ABOUT WHAT HAVING PROFILES, BUT NO TRACING WOULD BE GOOD FOR]...., we recommend setting up tracing, which can complement profiling in valuable ways [WOULD BE GOOD TO ADD A QUICK EXAMPLE OF VALUABLE WAYS TRACING + PROFILING CAN WORK TOGETHER].

If you've configured tracing and are sending us tracing data, but still see this banner, please reach out to our [customer support](https://sentry.zendesk.com/hc/en-us/).
```
Would be awesome if we could add a screenshot with the "look at aggregate flamegraph" banner showing.